openfisca_us/variables/gov/states/md/tax/income/credits/md_cdcc.py

```diff
@@ -0,0 +1,35 @@
+from openfisca_us.model_api import *
+
+
+class md_cdcc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MD CDCC"
+    documentation = "Maryland Child and Dependent Care Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-716-for-child-care-or-dependent-care"
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        is_joint = filing_status == filing_status.possible_values.JOINT
+        agi = tax_unit("adjusted_gross_income", period)
+        p = parameters(period).gov.states.md.tax.income.credits.cdcc
+        # Eligibility is based on AGI.
+        eligible = agi <= p.eligibility.agi_cap[filing_status]
+        # Maximum is a percent of federal.
+        max_cdcc = p.percent * tax_unit("cdcc", period)
+        # Phases out based on filing status.
+        phase_out_start = p.phase_out.start[filing_status]
+        excess = max_(0, agi - phase_out_start)
+        phase_out_increment = p.phase_out.increment[filing_status]
+        # TODO: Floor.
+        phase_out_increments = excess / phase_out_increment
```

Comment:
I think this should be a staircase, rather than a ramp, right?

Reply:
yes, added a floor statement